src/components/TasksTable/index.jsx

```diff
@@ -248,12 +248,26 @@ export default class TasksTable extends Component {
   handleDrawerOpen = ({ target: { name } }) => {
     memoizeWith(
       name => name,
-      name =>
+      async (name, onBugInfoClick) => {
+        const item = this.props.items.find(item => item.summary === name);
+
+        if (item.url.indexOf('bugzilla') === -1) {
```

Comment:
This will not work if the github url contains the the string bugzilla (e.g., https://github.com/mozilla/bugzilla). You can do:

```js
item.url.includes('bugzilla.mozilla.org');
```